2024/Day 02/1.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+with open('data.txt') as f:
```

Comment:
Ik zou dit wrappen in

```
def main():
   ...


if __name__ == "__main__":
    main()
```

Zodat het duidelijker is hoe de code loopt. Dan is het ook duidelijk waar data vandaan komt, en dan gebruik je global scope minder.